src/pudl/analysis/eia_ferc1_record_linkage.py

```diff
@@ -939,24 +939,31 @@ def check_match_consistency(
     train_df: pd.DataFrame,
     match_set: Literal["all", "overrides"] = "all",
 ) -> pd.DataFrame:
-    """Check how consistent matches are across time.
+    """Check how consistent FERC-EIA matches are with FERC-FERC matches.
+
+    We have two record linkage processes: one that links FERC plant records across time,
+    and another that links FERC plant records to EIA plant-parts. This function checks
+    that the two processes are as consistent with each other as we expect.  Here
+    "consistent" means that each FERC plant ID is associated with a single EIA plant
+    parts ID across time. The reverse is not necessarily required -- a single EIA plant
+    part ID may be associated with various FERC plant IDs across time.
 
     Args:
         connects_ferc1_eia: Matches of FERC1 to EIA.
         train_df: training data.
         match_set: either ``all`` - to check all of the matches - or ``overrides`` - to
-            check just the overrides. Default is'``all``. The overrides are less
+            check just the overrides. Default is ``all``. The overrides are less
             consistent than all of the data, so this argument changes the consistency
             threshold for this check.
     """
     # these are the default
-    consistency = 0.75
-    consistency_one_cap_ferc = 0.85
+    expected_consistency = 0.74
+    expected_uniform_capacity_consistency = 0.85
```

Comment:
It is surprising to me given how many parts of the overall FERC-FERC matching process changed, that we would end up within a fraction of a percent of the old threshold.

I suspect that `capacity_mw` should be weighted more heavily in the FERC-FERC matching, since a big change (like, 10% or more) in the capacity *probably* means that which chunk of plant is being reported has changed.